### ISSUE TITLE
"raw::Slice does not name a structure"

### ISSUE THREAD

**Issue by Ketsuban** [gz/rust-cpuid#4]
I updated my nightly compiler and libcore to 2016-06-03 and am now being met with this error.

```
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:435:25: 435:35 error: `raw::Slice` does not name a structure [E0422]
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:435             let slice = raw::Slice { data: brand_string_start, len: 3*4 };
                                                                                                                    ^~~~~~~~~~
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:435:25: 435:35 help: run `rustc --explain E0422` to see a detailed explanation
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:2097:29: 2097:39 error: `raw::Slice` does not name a structure [E0422]
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:2097                 let slice = raw::Slice { data: brand_string_start, len: 3*4*4 };
                                                                                                                         ^~~~~~~~~~
/home/thomas/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-2.0.0/src/lib.rs:2097:29: 2097:39 help: run `rustc --explain E0422` to see a detailed explanation
```

**Edit**: Okay, now I've had a chance to look at this in more detail I see the problem was solved in a commit two days ago, but hasn't made it onto crates.io yet. Could you push a point release so those of us using raw-cpuid only as a dependency (of the x86 crate, in this case) can take advantage of it?

**Comment by gz** [comment]
Hi, Sure I will be able to release a new version this Wednesday (unfortunately I don't have a laptop until then).

**Comment by Ketsuban** [comment]
Thanks very much. In the meantime, I've applied the patch to my local copy of the library (in the Cargo registry).

**Comment by gz** [comment]
Released new cpuid version 2.0.1 on Cargo.
